crates/engine/tree/src/persistence.rs

```diff
@@ -145,15 +149,18 @@ impl<DB: Database> PersistenceService<DB> {
     ///
     /// The [`update_transaction_meta`](Self::update_transaction_meta) method should be called
     /// after this, to update the checkpoints for headers and block bodies.
-    #[instrument(level = "trace", skip(self), target = "engine")]
-    fn write_transactions(&self, block: Arc<SealedBlock>) -> ProviderResult<(u64, U256)> {
+    #[instrument(level = "trace", skip(self, provider_rw), target = "engine")]
```

Comment:
this should only trace the blocks num_hash, because this is very noisy otherwise